source/sessions/snapshot-sessions.rst

```diff
@@ -130,19 +130,19 @@ Note that the ``snapshot`` property is optional. The default value of
 this property is false.
 
 Snapshot reads and causal consistency are mutually exclusive. Therefore if ``snapshot`` is set to true,
-``causalConsistency`` property is set to false. Client MUST throw an Error if both ``snapshot`` and ``causalConsistency`` are set to true.
-Snapshot reads are supported both on primaries and secondaries.
+``causalConsistency`` must be false. Client MUST throw an error if both ``snapshot`` and ``causalConsistency`` are set to true.
+Snapshot reads are supported on both primaries and secondaries.
 
-ClientSession changes
+ClientSession Changes
 =====================
 
 Transaction are not allowed with snapshot sessions.
-Calling ``session.startTransaction(options)`` on snapshot session should raise an error.
+Calling ``session.startTransaction(options)`` on a snapshot session SHOULD raise an error.
 
-ReadConcern changes
+ReadConcern Changes
```

Comment:
Should the case match the case used in [sessions spec](https://github.com/mongodb/specifications/blob/master/source/sessions/driver-sessions.rst) ?

Reply:
I'll revert this. IMO, there are wild inconsistencies across our specs when it comes to headings and [title capitalization](https://en.wikipedia.org/wiki/Title_case).

Reply:
```suggestion
ReadConcern changes
```